dp_wizard/shiny/panels/results_panel/__init__.py

```diff
@@ -255,43 +246,13 @@ def download_code_ui():
                     but does not contain any data or analysis results.
                     """
                 ),
+                responsive=False,
```

Comment:
Since this is now a two column UI, we want this box to span the full width of the card.